### PR TITLE
Fix gocardless bank "Hanseatic Bank" (access_valid_for_days max 89)

### PR DESCRIPTION
**Pull request by Froghut**

### CODE REVIEW

File: src/app-gocardless/bank-factory.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ import SpkMarburgBiedenkopfHeladef1mar from './banks/spk-marburg-biedenkopf-hela
 import SpkWormsAlzeyRiedMalade51wor from './banks/spk-worms-alzey-ried-malade51wor.js';
 import SwedbankHabaLV22 from './banks/swedbank-habalv22.js';
 import VirginNrnbgb22 from './banks/virgin_nrnbgb22.js';
+import HanseaticBank from './banks/HANSEATIC_HSTBDEHH.js';
```
Comment (matt-fidd) [inline src/app-gocardless/bank-factory.js:31]: Could this be moved so it's in alphabetical order please?

File: src/app-gocardless/bank-factory.js [hunk 2]
```diff
@@ -59,6 +60,7 @@ export const banks = [
   SpkWormsAlzeyRiedMalade51wor,
   SwedbankHabaLV22,
   VirginNrnbgb22,
+  HanseaticBank,
```
Comment (matt-fidd) [inline src/app-gocardless/bank-factory.js:63]: Same here

**Review by matt-fidd — CHANGES_REQUESTED** [review]
Thanks for this! Just a couple of comments before it can be merged

File: src/app-gocardless/bank-factory.js (hunk 2 above)
Comment (Froghut) [reply src/app-gocardless/bank-factory.js]: Ah sorry, totally missed that that is in alphabetical order

**Review by matt-fidd — APPROVED** [review]
Thank you!

**Pull request merged**